test/framework/src/main/java/org/elasticsearch/cluster/ESAllocationTestCase.java

```diff
@@ -66,6 +71,19 @@ public Long getShardSize(ShardRouting shardRouting) {
         }
     };
 
+    public static final WriteLoadForecaster TEST_WRITE_LOAD_FORECASTER = new WriteLoadForecaster() {
```

Comment:
nit: this is the same as `ESAllocationTestCase#SIMULATION_WRITE_LOAD_FORECASTER` (introduced since this PR was started). I prefer the name you use here, but let's rename the existing one rather than making a new one.